UNO_Server/Models/Game.cs

```diff
@@ -214,10 +210,12 @@ public bool CanCardBePlayed(Card playerCard)
 
 		private void NotifyAllObservers(Card card)
 		{
-			foreach (var item in observers)
-			{
-				item.Notify(card);
-			}
+            var iterator = gameWatcher.GetIterator();
+            while (iterator.HasNext)
+            {
+                Observer observer = (Observer)iterator.Next;
+                observer.Notify(card);
+            }
```

Comment:
Weird spacing again

Reply:
put all cardsCounter.AddCard methods in here and not all over the place

Reply:
hot fix incoming